server/routes/index.ts

```diff
@@ -35,5 +37,9 @@ export default function routes(services: Services): Router {
   get('/nomis-report-config/prisoner-involvement-roles.csv', downloadNomisConfigRoutes.prisonerInvolvementRoles)
   get('/nomis-report-config/prisoner-involvement-outcome.csv', downloadNomisConfigRoutes.prisonerInvolvementOutcome)
 
+  router.use('/create-incident', createIncidentRouter)
+  router.use('/change-incident/:id/', changeIncidentRouter)
+  router.use('/report/:incidentId/add-prisoner/:prisonerId', changeIncidentRouter)
```

Comment:
i feel that this needs the same `config.environment !== 'prod'` guard as other routes. what do you think?

Reply:
good point, I will move these now